### PR TITLE
Timer

### PR DESCRIPTION
**Pull request by stephanieea**
Added a css file.
Added a timer with limited functionality (increments up by 10 seconds).
⌛️ 

@gprocell927 @apsitos

### CODE REVIEW

File: renderer.js [hunk 1]
```diff
@@ -26,3 +28,29 @@ const validateUrl = (link) => {
...
+}
+
+  // Initial time, hardcoded
+  let currentTime = 255;
+
+  // Print out the time
+  timerDiv.innerHTML = secondsToTime(currentTime);
+
+  // Execute every second
+  let timer = setInterval(() => {
```
Comment (gprocell927) [inline renderer.js]: I was comparing this code to the code in the Wes Bos tutorial which used some more 'rithmetic:
`countdown = setInterval(() => {
    const secondsLeft = Math.round((then - Date.now()) / 1000);
    // check if we should stop it!
    if(secondsLeft < 0) {
      clearInterval(countdown);
      return;
    }
    // display it
    displayTimeLeft(secondsLeft);
  }, 1000);`

**Review by gprocell927 — COMMENTED** [review]
I like the use of moment for formatting. I guess the question is if we truly need that. Again, I'm just going off of the one timer tutorial that I have done before. Maybe I misunderstood what the goal was? ⏰

**Comment by stephanieea** [comment]
Hmm, when I looked back at JavaScript30, if that's the tutorial you remember, it was more for moving a round clock rather than setting up a timer.  I liked the idea of using moment because it seemed cleaner than setting it up with vanilla JavaScript/jQuery but if we don't want to use it, I can definitely redo it that way?  Let me know which you prefer @apsitos @gprocell927 .

Also, do we want the timer to have an input field where they input the time or just increment up based on a plus?  For whatever reason when I was making this I added the plus and - buttons because I was thinking about my kitchen microwave timer but an input field makes more sense probably, no?  ⌚️

**Comment by apsitos** [comment]
I also really like using Moment. I think the timer should increment by minutes. Using a button is just fine!

**Pull request closed without merging**